### PR TITLE
Minor grammar and spelling fixes in docs

### PR DESCRIPTION
**Pull request by vivshaw**
# Description

Hello, I've just completed an alpha test, and I figured I'd help tidy up the docs a little along the way! Here are some edits that fix a few spelling or grammar mistakes in the tutorial. Additionally, I found two steps that had an error in them- I corrected those and will call them out in self-review below.

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

File: web/docs/tutorials/todo-app/auth.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Now, we can again run
 wasp start
 ```
 
-Try going to `/` in our web app -> it will now ask you to log in, and if you follow the link, you will end up at `/login`.
+Try going to `/` in our web app. It will now redirect you to `/login`, where you'll be asked to authenticate.
```
Comment (vivshaw) [inline web/docs/tutorials/todo-app/auth.md:164]: It seems you no longer need to follow any link. When I followed the tutorial, I found that I was automatically redirected to `/login`. So I updated this line accordingly.👍

File: web/docs/tutorials/todo-app/dependencies.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ app TodoApp {
 }
 ```
 
-Run (if it is already running, stop it first and then run it again)
+Run
 ```shell-session
 wasp start
 ```
-to have Wasp download and install new dependency (that happens on start of `wasp start`).
+to have Wasp download and install the new dependency. If `wasp start` is already running, Wasp will detect the dependency change, and restart automatically.
```
Comment (vivshaw) [inline web/docs/tutorials/todo-app/dependencies.md:28]: The docs said to start and re-run `wasp start`, but when I followed the tutorial, I found that `wasp start` helpfully detected the dependency change and restarted itself. So I updated the docs to say that!

File: web/docs/tutorials/todo-app/the-end.md [hunk 3]
```diff
@@ -13,14 +13,14 @@ We did it! For all those that followed the instructions closely and created "Bui
 
 You can check out the whole code of the app that we just built [here](https://github.com/wasp-lang/wasp/tree/main/examples/tutorials/TodoApp).
 
-If you are interested in what is Wasp actually generating in the background, you can check `.wasp/out/` directory in your project.
+If you are interested in what is Wasp actually generating in the background, you can check the `.wasp/out/` directory in your project.
 
 ## Where next?
 
-Well, you could check the "Language" section of the docs for more details on specific parts of Wasp.  
+Well, you could check [the "Language" section](language/overview.md) of the docs for more details on specific parts of Wasp.
```
Comment (vivshaw) [inline web/docs/tutorials/todo-app/the-end.md:20]: Added a direct link, so folks can get there more easily!

File: web/docs/tutorials/todo-app/dependencies.md (hunk 2 above)
Comment (Martinsos) [reply web/docs/tutorials/todo-app/dependencies.md:28]: Ah true we added that recently :)!

**Review by Martinsos — APPROVED** [review]
This is awesome @vivshaw ! Thanks so much, both for fixing grammar mistakes and for making it clearer for future reads as to what should be happening in the app. I am glad that the discrepancies you found were mostly due to new functionality getting released :D!
Merging this one, looks great!

**Pull request merged**